### ISSUE TITLE
v6 Field doesn't rerender when handleSubmit triggered

### ISSUE THREAD

**Issue by vslinko** [erikras/redux-form#1066]
I have following form:

```typescript
export default class ExampleForm extends React.Component {
  render() {
    return (
      <form onSubmit={this.props.handleSubmit}>
        <Field
          name="name"
          component={(props) => (
            <TextField
              disabled={this.props.submitting}
              value={props.value}
              onValueChange={props.onChange}
            />
          )}
        />
```

Field doesn't updates after form submission and after end of form submission.
Am I doing something wrong?

**Comment by erikras** [comment]
Yes, you are doing something wrong. (it was right before, but now is wrong) [Read this](https://github.com/erikras/redux-form/releases/tag/v6.0.0-alpha.14) and update to 6a14.

**Comment by vslinko** [comment]
But how to pass extra props through `Field`?

**Comment by erikras** [comment]
Yes, through `Field`. Any prop you give to `Field` will be passed on to the `component`.

**Comment by vslinko** [comment]
Hmm. It's ok for JavaScript, but it hard to declare using TypeScript type system.
Why don't use one prop for passing extra props `<Field name="a" component={x} props={{a: 1}} />`?
I think it should be easier for everything: propTypes, typescript, documentation.
Also, what if I need pass `name` property?

**Comment by erikras** [comment]
Seems like it'd be okay to make `'props'` a special prop name that would get destructured into the other props.

**Comment by erikras** [comment]
Published fix as [`6.0.0-alpha-15`](https://github.com/erikras/redux-form/releases/tag/v6.0.0-alpha.15).
